### PR TITLE
Added a new column and updated existing template categories

### PR DESCRIPTION
**Pull request by jzbahrai**
# Summary | Résumé

Added a new column to the TemplateCategory table called "sms_sending_vehicle". Added reasonable defaults.
Updated priority type for old template categories.

SMS_sending_vehicle is currently determined here: https://docs.google.com/document/d/1xhBHjVywBgZTk2A76B7hJOSj0RjpR5ttu6cGDBI98PQ/edit
we can change it through a another migration once further discussion has taken place

* https://app.zenhub.com/workspaces/notify-planning-614b3ad91bc2030015ed22f5/issues/gh/cds-snc/notification-planning/1609

# QA
```
notification_api=# select name_en, sms_sending_vehicle from template_categories;
         name_en          | sms_sending_vehicle 
--------------------------+---------------------
 Authentication           | short_code
 Automatic reply          | short_code
 Decision                 | short_code
 Reminder                 | short_code
 Request                  | short_code
 Status update            | short_code
 High Category (Priority) | short_code
 Alert                    | long_code
 Information blast        | long_code
 Test                     | long_code
 Low Category (Bulk)      | long_code
 Medium Category (Normal) | long_code
 ```

# Reviewer checklist | Liste de vérification du réviseur

- [ ] This PR does not break existing functionality.
- [ ] This PR does not violate GCNotify's privacy policies.
- [ ] This PR does not raise new security concerns. Refer to our GC Notify Risk Register document on our Google drive.
- [ ] This PR does not significantly alter performance.
- [ ] Additional required documentation resulting of these changes is covered (such as the README, setup instructions, a related ADR or the technical documentation).

> ⚠ If boxes cannot be checked off before merging the PR, they should be moved to the "Release Instructions" section with appropriate steps required to verify before release. For example, changes to celery code may require tests on staging to verify that performance has not been affected.

### CODE REVIEW

File: migrations/versions/0456_update_template_categories.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+
+def upgrade():
+    op.add_column(
+        "template_categories", sa.Column("sms_sending_vehicle", sa.String(length=255), server_default="long_code", nullable=False)
+    )
+
+    # Update the generic categories
+    op.execute(
+        "UPDATE template_categories SET sms_process_type = 'bulk', email_process_type = 'bulk' WHERE id = '{}'".format(
```
Comment (sastels) [inline migrations/versions/0456_update_template_categories.py:36]: I wish we could rename `bulk`, `normal`, and `priority` to `low`, `medium` and `high` everywhere but I know it's 💯 out of scope for this PR.

Comment (jzbahrai) [reply migrations/versions/0456_update_template_categories.py:41]: ya :(

File: migrations/versions/0456_update_template_categories.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+CAT_REMINDER_ID = "edb966f3-4a4c-47a4-96ab-05ff259b919c"
+CAT_REQUEST_ID = "e0b8fbe5-f435-4977-8fc8-03f13d9296a5"
+CAT_STATUS_ID = "55eb1137-6dc6-4094-9031-f61124a279dc"
+CAT_TEST_ID = "7c16aa95-e2e1-4497-81d6-04c656520fe4"
+
+SHORT_CODE_CATS = (HIGH_CATEGORY_ID, CAT_AUTH_ID, CAT_AUTO_ID, CAT_DECISION_ID, CAT_REMINDER_ID, CAT_REQUEST_ID, CAT_STATUS_ID)
+LONG_CODE_CATS = (LOW_CATEGORY_ID, MEDIUM_CATEGORY_ID, CAT_ALERT_ID, CAT_INFO_ID, CAT_TEST_ID)
+
+sms_options = ("short_code", "long_code")
+sms_sending_vehicle = sa.Enum(*sms_options, name="sms_sending_vehicle")
```
Comment (jzbahrai) [inline migrations/versions/0456_update_template_categories.py:29]: set the sending vehicles as Enums

Comment (andrewleith) [reply migrations/versions/0456_update_template_categories.py:29]: Should we get `DEFAULT_TEMPLATE_CATEGORY_LOW.*` from `config.py`?

Comment (jzbahrai) [reply migrations/versions/0456_update_template_categories.py:29]: i think its fine, we have it hardcoded by the time this migration is run as it was introduced in the previous migration

**Review by andrewleith — APPROVED** [review]
LGTM

**Pull request merged**